sources/packages/backend/libs/integrations/src/sfas-integration/sims-to-sfas.processing.service.ts

```diff
@@ -41,25 +41,51 @@ export class SIMSToSFASProcessingService {
     const simsToSFASStudents = new SIMSToSFASStudents();
     processSummary.info("Get all the students with updates.");
     // Get all the students with updates in student related data.
-    // TODO: SIMS to SFAS - Get all the students with updates in application and restriction related data.
     const studentIds = await this.simsToSFASService.getAllStudentsWithUpdates(
       modifiedSince,
       modifiedUntil,
     );
 
+    const applicationRecordsPromise =
+      this.simsToSFASService.getAllApplicationsWithUpdates(
+        modifiedSince,
+        modifiedUntil,
+      );
+
+    const restrictionRecordsPromise =
+      this.simsToSFASService.getAllRestrictionWithUpdates(
+        modifiedSince,
+        modifiedUntil,
+      );
+
```

Comment:
Please add some info logs here. 

processSummary.info("Get all the applications and restrictions with updates.");